spec/lib/sarif/bundle_audit_spec.rb

```diff
@@ -135,4 +138,30 @@
       end
     end
   end
+
+  describe 'sarif diff' do
+    context 'git diff support' do
+      it 'should find code in git diff' do
+        git_diff_file = 'spec/fixtures/sarifs/diff/git_diff_7.txt'
+        snippet = 'helloworld'
+        git_diff = File.read(git_diff_file)
+        new_lines_in_git_diff = Sarif::BaseSarif.new_lines_in_git_diff(git_diff)
+        r = Sarif::BundleAuditSarif.snippet_possibly_in_git_diff?(snippet, new_lines_in_git_diff)
+        expect(r).to be true
+        snippet = 'bye'
+        r = Sarif::BundleAuditSarif.snippet_possibly_in_git_diff?(snippet, new_lines_in_git_diff)
+        expect(r).to be true
+
+        # should not match part of the package name
+        snippet = 'hello'
+        r = Sarif::BundleAuditSarif.snippet_possibly_in_git_diff?(snippet, new_lines_in_git_diff)
+        expect(r).to be false
+
+        # should not match package that was in git diff but not added with this commit
```

Comment:
Please write this as a separate unit test

it 'should not match package that was in git diff but not added with this commit' do
end